app/controllers/idv/hybrid_mobile/socure/document_capture_controller.rb

```diff
@@ -47,7 +48,27 @@ def show
         end
 
         def update
-          render plain: 'stub to ensure Socure callback exists and the route works'
+          result = handle_stored_result
+
+          if result.success?
+            redirect_to idv_ssn_url
+          else
+            redirect_to idv_socure_document_capture_url
+          end
```

Comment:
```suggestion
          redirect_to idv_hybrid_mobile_socure_document_capture_url
```

should we instead be redireting to the hybrid mobile url here ...


also, seems like we should be using this [confirm_document_capture_needed filter](https://github.com/18F/identity-idp/blob/22e2ab288a7b387ef100da380e819c7c75203997/app/controllers/idv/hybrid_mobile/document_capture_controller.rb#L12)? 🤔  ...
if so, can we move this filter in to a concern for use in both hybrid mobile document capture controllers?

Reply:
We do want to redirect to SSN on success, but you're right that we want to go back to hybrid mobile on failure. Changed.